api-reference/endpoint/chat/completions.mdx

```diff
@@ -1,99 +1,297 @@
+<ParamField body="response" type="Object" required="true">
+  Defines the desired response format and options.
+  <Expandable title="properties">
+    <ParamField path="format" type="string" required="true">
+      The format of the response. Must be one of `text` or `json_object`.
+      Default is `text`.
+    </ParamField>
+    <ParamField path="stream" type="boolean">
+      Whether the response should be streamed. Default is `false`.
+    </ParamField>
+    <ParamField path="tools" type="array">
+      An array of tools to be used for processing the response.
+      <Expandable title="properties">
+        <ParamField path="type" type="string" required="true">
+          The type of the tool. Must be one of 'function'.
         </ParamField>
-
-        <ParamField path="name" type="string" required="true">
-            The name of the model. This supports all the models available on the specified platforms.
+        <ParamField path="function" type="Object">
+          The function to be used for processing the response.
+          <Expandable title="properties">
+            <ParamField
+              path="name"
+              type="string"
+              required="true"
+              max="64"
+              regex="^[a-zA-Z0-9_-]+$"
+            >
+              The name of the function. Must be a string with a maximum length
+              of 64 and match the regex /^[a-zA-Z0-9_-]+$/.
+            </ParamField>
+            <ParamField path="description" type="string">
+              The description of the function.
+            </ParamField>
+            <ParamField path="parameters" type="Object">
+              The parameters of the function.
+              <ParamField path="<parameter>" type="any">
+                The parameter name and type.
+              </ParamField>
+            </ParamField>
+          </Expandable>
         </ParamField>
-    </Expandable>
+      </Expandable>
+    </ParamField>
+    <ParamField path="preferred_tool" type="string">
+      The preferred tool to be used for processing the response. Must be one of
+      the tools in the tools list.
+    </ParamField>
+  </Expandable>
 </ParamField>
 
-<ParamField path="options" type="Object">
-    You can pass additional parameters to be used by Gateway API.
+<ParamField body="options" type="Object">
+  Contains any additional parameters to be used by Gateway API.
 </ParamField>
 
-
 ## Chat Completion Response
 
+This response will contain the model's response to the chat completion request.
+
+<ResponseField name="tryId" type="string">
+  The unique identifier of the chat completion request.
+</ResponseField>
+
+<ResponseField name="status" type="Object">
+  The status of the chat completion response.
+  <Expandable title="properties">
+    <ResponseField name="type" type="string">
+      The type of the status. Can be one of `ok`, `platform_error`, `error`.
+    </ResponseField>
+    <ResponseField name="code" type="number">
```

Comment:
This response is common for all apis. It should be there in another page in docs. Then there should be a link from here mentioning that.